addons/l10n_din5008/report/din5008_base_document_layout.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_wizard_preview" inherit_id="web.report_invoice_wizard_preview">
+        <xpath expr="//t[@t-set='address']" position="after">
+            <t t-set="din5008_document_information">
+                <div class="information_block" t-if="not o and not docs">
+                    <table>
+                        <tr>
+                            <td>Invoice No.:</td>
+                            <td>INV/2021/12345</td>
+                        </tr>
+                        <tr>
+                            <td>Invoice Date:</td>
+                            <td>14/04/2023</td>
```

Comment:
Fake data for the configure layout, i could make a field with today as default, but since it's a preview do we care ?
same for the Due Date and delivery Date

Reply:
Then you'd need to since the invoice number, the due date, etc... (which is not really the case here already)

Reply:
I think it's missing a part of the sentence, but i will add the fields to be sure